web/web_test.go

```diff
@@ -4,3 +4,60 @@
 // LICENSE file in the root directory of this source tree.
 
 package web
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/helpers"
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/iam"
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/resources"
+	"github.com/Azure/azure-sdk-for-go/services/web/mgmt/2016-09-01/web"
+)
+
+var (
+	appServicePlanName = "web-appserviceplan-go-samples" + helpers.GetRandomLetterSequence(10)
+	siteName           = "web-site-go-samples" + helpers.GetRandomLetterSequence(10)
+)
+
+func TestMain(m *testing.M) {
+	flag.StringVar(&appServicePlanName, "appServicePlanName", appServicePlanName, "Optionally provide a name for the App Service Plan to be created.")
+	flag.StringVar(&siteName, "siteName", siteName, "Optionally provided a name for the Site to be created.")
+
+	err := iam.ParseArgs()
+	if err != nil {
+		log.Fatalln("failed to parse IAM args")
+	}
+
+	os.Exit(m.Run())
+}
+
+func ExampleWeb_DeployAppForContainer() {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute*10)
+	defer cancel()
+
+	helpers.SetResourceGroupName("WebAppForContainers")
+	_, err := resources.CreateGroup(ctx, helpers.ResourceGroupName())
+	if err != nil {
+		fmt.Println("failed to create resource group: ", err)
+		return
+	}
+	defer resources.Cleanup(ctx)
+
+	var configResource web.SiteConfigResource
```

Comment:
Would it be better to use `:=` and have this defined and initialized in the next line?